### PR TITLE
#170947647 Integrate HoundCI and Add Its Badge

### PR DESCRIPTION
**Pull request by Moise1**
#### What does this PR do?
This PR integrates HoundCI and its badge 

#### Description of Task to be completed?
Successfully integrating HoundCI and its badge 

#### How should this be manually tested?
- run `git clone https://github.com/andela/knights-bn-backend.git` in your terminal
- run `cd knights-bn-backend` in your terminal
-  Check the README.md file to see the Hound's badge 
- Check the .hound.yml file to see its configuration. 

- Raise a Pull Request then, Hound reviews code in your GitHub pull requests. If Hound finds any issues it will comment inline on your pull request.

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#170947647](https://www.pivotaltracker.com/n/projects/2431452)

#### Screenshots (if appropriate)
N/A

### CODE REVIEW

File: .hound.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+eslint:
+  enabled: true
+  config_file: .eslintrc
+
```
Comment (Niyonsengaeric) [inline .hound.yml]: new line

**Review by Niyonsengaeric — CHANGES_REQUESTED** [review]

**Comment by T2Wil** [comment]
hey @Moise1 ,
Squash the 2 commits into one commit

**Comment by EugeneMunya** [comment]
Remember to resolve the conflict

**Review by Niyonsengaeric — APPROVED** [review]

**Review by MuhanguziDavid — APPROVED** [review]
LGTM

**Pull request merged**